lib/events.js

```diff
@@ -508,3 +514,22 @@ function once(emitter, name) {
     emitter.once(name, eventListener);
   });
 }
+
+function bindEventEmitterFunction(name, target) {
+  if (typeof name !== 'string')
+    throw new ERR_INVALID_ARG_TYPE('name', 'string', name);
```

Comment:
Since this is an internal-only function that is only ever called with string literals, is this code even reachable? If not, it should be removed or replaced with `assert()` (from `lib/internal/assert` to keep it lean).